hash/megadriv.xml

```diff
@@ -3006,6 +3006,58 @@ Crashes after EA logo, requires better [VDP] irq handling
 	</software>
 
 
+	<software name="felixjr">
+		<description>Fix-it Felix Jr.</description>
+		<year>2014</year>
+		<publisher>Airwalk Studios</publisher>
+		<part name="cart" interface="megadriv_cart">
+			<dataarea name="rom" width="16" endianness="big" size="601870">
+				<rom name="fix-it felix jr (final)" size="601870" crc="424ff451" sha1="2bad4a0db0f95a7588e75f344c25427a6e83522b" />
+			</dataarea>
+		</part>
+	</software>
+
+
+	<!-- Another, 1.12, version is known to have existed between this and the final version
+		 See: https://www.youtube.com/watch?v=_1uGkfw5ky8 -->
+	<software name="felixjro" cloneof="felixjr">
+		<description>Fix-it Felix Jr. (v1.0)</description>
```

Comment:
Please use a mnemonic suffix in the short name for the clone to indicate that it’s version 1.0